src/components/ChangePlan/PlanCalculator.styles.js

```diff
@@ -11,3 +11,9 @@ export const PlanTabContainer = styled.section`
     }
   }
 `;
+
+export const HiddenTooltip = styled.div`
+  .disabled {
```

Comment:
How about naming this class hidden? since you are hiding this element and not disabling it.